includes/definitions/fujifilmprinter.yaml

```diff
@@ -0,0 +1,29 @@
+os: fujifilmprinter
+text: 'Fujifilm Printer'
+type: printer
+icon: fujifilm
+over:
+    - { graph: device_toner, text: Toner }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.297.1.11.93.2.1
+discovery_modules:
+    printer-supplies:   true
+    ports-stack: false
+    arp-table: false
+    bgp-peers: false
+    vlans: false
+    ucd-diskio: false
+    stp: false
+    fdb-table: false
+    mpls: false
+poller_modules:
+    printer-supplies:   true
+    arp-table: false
+    bgp-peers: false
+    vlans: false
+    ucd-diskio: false
+    stp: false
+    mpls: false
+    ucd-mib: false
+    ospf: false
```

Comment:
dont disable modules if they dont cause issues.

Reply:
Most poller modules do a very quick check to see if they should poll based on the discovery module.  So, mostly it is enough to only disable the discovery module ...  and the discovery module runs only once every 6 hours so it should  not be very disruptive.